### PR TITLE
Adds multiple auto-pruning policies info

### PR DESCRIPTION
**Pull request by stevsmit**

### CODE REVIEW

File: modules/managing-namespace-auto-pruning-policies.adoc [hunk 1]
```diff
@@ -232,13 +236,43 @@ $ curl -X POST -H "Authorization: Bearer <access_token>" -H "Content-Type: appli
...
+----
+{"uuid": "ebf7448b-93c3-4f14-bf2f-25aa6857c7b0"}
+----
+
+. You can update your organization's auto-prune policy by using the link:https://docs.redhat.com/en/documentation/red_hat_quay/{producty}/html-single/red_hat_quay_api_guide/index#updateorganizationautoprunepolicy[`PUT /api/v1/organization/{orgname}/autoprunepolicy/{policy_uuid}`] command. For example:
 +
 [source,terminal]
 ----
-{"detail": "Policy for this namespace already exists, delete existing to create new policy", "error_message": "Policy for this namespace already exists, delete existing to create new policy", "error_type": "invalid_request", "title": "invalid_request", "type": "http://<quay-server.example.com>/api/v1/error/invalid_request", "status": 400}
+$ $ curl -X PUT   -H "Authorization: Bearer <bearer_token>"   -H "Content-Type: application/json"   -d '{
```
Comment (hu-weihua) [inline modules/managing-namespace-auto-pruning-policies.adoc:267]: There are two `$` here.

File: modules/managing-namespace-auto-pruning-policies.adoc [hunk 2]
```diff
@@ -191,6 +191,10 @@ image:auto-prune-policies-page.png[Auto-Prune Policies page]
 
 . Select the desired number of tags to keep. By default, this is set at *20* tags. For this example, the number of tags to keep is set at *3*.
 
+. Optional. To prune only tags that match the given regex pattern, click the *Tag pattern* box and select *match*. In the regex box, enter a pattern to match tags against. For example, to automatically prune all `test` tags, enter `^test.*`. To prune a specific image, you can enter `^test1*`.
```
Comment (hu-weihua) [inline modules/managing-namespace-auto-pruning-policies.adoc:194]: I am not sure if I understand the sentence  `To prune a specific image, you can enter `^test1*` correctly.`,  If we say `a specific image`,  the regular expression should be `^test1$`,  right?

File: modules/managing-namespace-auto-pruning-policies.adoc [hunk 3]
```diff
@@ -397,6 +431,10 @@ $ podman push <quay-server.example.com>/<organization_name>/<repository_name>:te
...
```
Comment (hu-weihua) [inline modules/managing-namespace-auto-pruning-policies.adoc:434]: If we say a specific image, the regular expression should be ^test1$, right?

File: modules/api-policy-createOrganizationAutoPrunePolicy.adoc [hunk 4]
```diff
@@ -30,9 +30,13 @@ The policy configuration that is to be applied to the user namespace
 |===
 |Name|Description|Schema
 |**method** + 
-_required_|The method to use for pruning tags (number_of_tags, creation_date)|string
+_optional_|The method to use for pruning tags (number_of_tags, creation_date)|string
```
Comment (Sunandadadi) [inline modules/api-policy-createOrganizationAutoPrunePolicy.adoc:33]: This is `required`. Cannot create a policy without selecting the method.

File: modules/api-policy-createOrganizationAutoPrunePolicy.adoc [hunk 5]
```diff
@@ -30,9 +30,13 @@ The policy configuration that is to be applied to the user namespace
...
 |**value** + 
-_required_|The value to use for the pruning method (number of tags e.g. 10, time delta e.g. 7d (7 days))|
+_optional_|The value to use for the pruning method (number of tags e.g. 10, time delta e.g. 7d (7 days))|
```
Comment (Sunandadadi) [inline modules/api-policy-createOrganizationAutoPrunePolicy.adoc:35]: This is `required` too.

File: modules/api-policy-createRepositoryAutoPrunePolicy.adoc [hunk 6]
```diff
@@ -33,6 +33,10 @@ The policy configuration that is to be applied to the user namespace
 _optional_|The method to use for pruning tags (number_of_tags, creation_date)|string
```
Comment (Sunandadadi) [inline modules/api-policy-createRepositoryAutoPrunePolicy.adoc:33]: same as above, if we create a policy, `method` and `value` are required.

File: modules/api-policy-createUserAutoPrunePolicy.adoc [hunk 7]
```diff
@@ -20,9 +20,13 @@ The policy configuration that is to be applied to the user namespace
 |===
 |Name|Description|Schema
 |**method** + 
-_required_|The method to use for pruning tags (number_of_tags, creation_date)|string
+_optional_|The method to use for pruning tags (number_of_tags, creation_date)|string
```
Comment (Sunandadadi) [inline modules/api-policy-createUserAutoPrunePolicy.adoc:23]: same as above. `method` and `value` are required.

File: modules/api-policy-updateOrganizationAutoPrunePolicy.adoc [hunk 8]
```diff
@@ -32,9 +32,13 @@ The policy configuration that is to be applied to the user namespace
 |===
 |Name|Description|Schema
 |**method** + 
-_required_|The method to use for pruning tags (number_of_tags, creation_date)|string
+_optional_|The method to use for pruning tags (number_of_tags, creation_date)|string
```
Comment (Sunandadadi) [inline modules/api-policy-updateOrganizationAutoPrunePolicy.adoc:35]: `method` and `value` are required.

File: modules/api-policy-updateOrganizationAutoPrunePolicy.adoc [hunk 9]
```diff
@@ -44,7 +48,7 @@ _required_|The value to use for the pruning method (number of tags e.g. 10, time
 [options="header", width=100%, cols=".^2a,.^14a,.^4a"]
 |===
 |HTTP Code|Description|Schema
-|204|Successful invocation|
+|200|Successful invocation|
```
Comment (Sunandadadi) [inline modules/api-policy-updateOrganizationAutoPrunePolicy.adoc:51]: update returns `204` on success.

File: modules/api-policy-updateUserAutoPrunePolicy.adoc [hunk 10]
```diff
@@ -30,9 +30,13 @@ The policy configuration that is to be applied to the user namespace
 |===
 |Name|Description|Schema
 |**method** + 
-_required_|The method to use for pruning tags (number_of_tags, creation_date)|string
+_optional_|The method to use for pruning tags (number_of_tags, creation_date)|string
```
Comment (Sunandadadi) [inline modules/api-policy-updateUserAutoPrunePolicy.adoc:33]: `method` and `value` are required.

File: modules/api-policy-updateUserAutoPrunePolicy.adoc [hunk 11]
```diff
@@ -42,7 +46,7 @@ _required_|The value to use for the pruning method (number of tags e.g. 10, time
 [options="header", width=100%, cols=".^2a,.^14a,.^4a"]
 |===
 |HTTP Code|Description|Schema
-|204|Successful invocation|
+|200|Successful invocation|
```
Comment (Sunandadadi) [inline modules/api-policy-updateUserAutoPrunePolicy.adoc:49]: successful update returns `204`.

File: modules/managing-namespace-auto-pruning-policies.adoc (hunk 2 above)
Comment (Sunandadadi) [reply modules/managing-namespace-auto-pruning-policies.adoc]: I'd write something like, To prune all tag names starting with `test`, enter `^test.*`

File: modules/rn_3_13_0.adoc [hunk 12]
```diff
@@ -46,7 +52,7 @@ The following enhancements have been made to the {productname} v2 UI.
 
 [id="new-quay-config-fields-313"]
 == New {productname} configuration fields
- 
+
```
Comment (Sunandadadi) [inline modules/rn_3_13_0.adoc:55]: nit: extra space

**Pull request merged**